pyproject.toml

```diff
@@ -18,22 +16,27 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
     "License :: OSI Approved :: BSD License",
     "Topic :: Software Development :: Libraries :: Python Modules",
-    "Topic :: Database :: Database Engines/Servers"
-]
-dependencies = [
-    "pyoxigraph~=0.3.14",
-    "rdflib>=6.3,<8.0"
+    "Topic :: Database :: Database Engines/Servers",
 ]
+dependencies = ["pyoxigraph~=0.3.14", "rdflib>=6.3,<8.0"]
 dynamic = ["version"]
 license = { text = "BSD-3-Clause" }
 name = "oxrdflib"
 readme = "README.md"
 requires-python = ">=3.7"
 
+
 [project.entry-points."rdf.plugins.store"]
-Oxigraph = "oxrdflib:OxigraphStore"
-OxMemory = "oxrdflib:OxigraphStore"
-OxSled = "oxrdflib:OxigraphStore"
+Oxigraph = "oxrdflib.store:OxigraphStore"
+OxMemory = "oxrdflib.store:OxigraphStore"
+OxSled = "oxrdflib.store:OxigraphStore"
+
+
+[project.entry-points."rdf.plugins.parser"]
+oxNTriples = "oxrdflib.parser:OxigraphNTriplesParser"
```

Comment:
Rdflib uses only lower case for the type names. I would maybe write `oxntriples` or `ox-ntriples` for consistency. WDYT?

Reply:
Would love second option but I think entry-point expects pythonic form (no hyphens), so the first option has to be selected.

Reply:
actually works with hyphen :)